ui/partials/perf/comparetable.html

```diff
@@ -15,7 +15,7 @@
     <tr ng-class="getCompareClasses(compareResult, 'row')" ng-repeat="compareResult in compareResults[testName] | orderBy: 'name'">
       <td class="test-title">{{compareResult.name}}
         <span ng-if="compareResult.detailsLink">
-          (<a ng-href="{{compareResult.detailsLink}}">subtests</a>)
+          (<a ng-href="{{compareResult.detailsLink}}" ng-show="compareResult.detailsLink">subtests</a>)
```

Comment:
This needs to be changed to something like:

    <span ng-show="compareResult.detailsLink">(<a ng-href="{{compareResult.detailsLink}}">subtests</a>)</span>

Otherwise we get a weird-looking () beside the counters test.